src/echo.ts

```diff
@@ -46,6 +51,8 @@ export default class Echo {
             this.connector = new NullConnector(this.options);
         } else if (typeof this.options.broadcaster == 'function') {
             this.connector = new this.options.broadcaster(this.options);
+        } else {
+            throw new Error('Broadcaster option not set when initialising Echo.');
```

Comment:
This is a bit nicer than the user calling a method on undefined as the connector has not been set.